autoload/vital/__vital__/VS/Vim/Window/Widget/MarkupContent.vimspec

```diff
@@ -0,0 +1,24 @@
+let s:expect = themis#helper('expect')
+let s:MarkupContent = vital#vital#import('VS.Vim.Window.Widget.MarkupContent')
+
+if !s:MarkupContent.is_available()
+  finish
+endif
+
+Describe vital#__vital__#VS#Vim#Window#Widget#MarkupContent
+
+  It should display markup contents
+    let l:markup_content = s:MarkupContent.new({
+    \   'maxwidth': float2nr(&columns * 0.3),
+    \   'maxheight': float2nr(&lines * 0.3),
+    \ })
+    call l:markup_content.open(1, 1, [
+    \   '```typescript',
+    \   'console.log(variable);',
+    \   '```',
+    \ ])
+    call l:markup_content.close()
+  End
```

Comment:
Hm... the border is complex to calculate position and size...

Reply:
there are multiple implementations for border.

https://github.com/nvim-lua/popup.nvim
https://github.com/lambdalisue/loupe.vim/blob/master/autoload/loupe/viewer/nvim.vim#L24-L105

neovim in vim-lsp doesn't support border so might be we can implement borders later? That way I can at least try converting in vim-lsp neovim to use this floatingwindow.